### ISSUE TITLE
Add accessibility ids for elements in new UI

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#14193]
In order to start process of writing new E2E tests new UI elements should have accessibility ids. 

Currently accessibility ids are missing for the following elements: 

1. All the main tabs: Communities, Chats, Wallet, Profile
2. Status switcher
<img width="447" alt="Appium 2022-10-20 14-35-47" src="https://user-images.githubusercontent.com/97245802/196938224-ea734c74-95dc-4411-a306-26f77472c45c.png">
3. Emoji items in message options
<img width="443" alt="Android Emulator - Pixel_XL_API_32:5554 2022-10-20 14-38-33" src="https://user-images.githubusercontent.com/97245802/196938526-f589b9fb-ea3e-49c0-b981-22b635d65dee.png">
4. Emoji below message counter
<img width="423" alt="Android Emulator - Pixel_XL_API_32:5554 2022-10-20 17-04-13" src="https://user-images.githubusercontent.com/97245802/196970755-b1d04d24-1d0f-4964-8fd1-f9eedaf16a07.png">
5. Reply message (for possibility alocate it apart from regular message)
<img width="425" alt="Android Emulator - Pixel_XL_API_32:5554 2022-10-20 17-34-58" src="https://user-images.githubusercontent.com/97245802/196979104-ca375824-1a27-4b26-9040-fca447db1217.png">

**Comment by Parveshdhull** [comment]
The switcher is replaced with Shell and the bottom tabs also changed, I will add the accessibility id for the new bottom tabs and main tabs and the jump-to button in https://github.com/status-im/status-mobile/pull/14099

**Comment by Parveshdhull** [comment]
Others are also small changes, will include in same PR
